### ISSUE TITLE
Multiple Repositories

### ISSUE THREAD

**Issue by geocentric** [NewProggie/SparkleShare-Android#69]
I have 2 git-repos, 1 to test and the second one for my "real" purposes. Everything was great with my test-Repo. No problems with syncing, performance or anything else. Great. ( Congrats & Thanks ) 

So i tried to add the work 'folder' (real repo), but i was not able to add a new folder in the android-app.  I deleted the test folder ('forget'). Everytime I tried to get access to the server (domain/ip-adress) and the 'real' repository, I am linked to the 'test'-repo-folder. :( The desktop-Application (Ubuntu) has no problems, works like a charm.  Any idea?

**Comment by NewProggie** [comment]
Did you setup Dashboard for your real repo as well?

**Comment by geocentric** [comment]
I included both reps in the config.js, from the beginning. I also tried to setup the repo in the Dashboard web-interface or in the android-app, but there was no way ( i found none ). 

Well i finally managed a workaround: 
1. i deleted all test-entries: the test git-repo, the second config-entry, the android-test-account.
2. and tried it again, with the 'real'-repo.
It works, at least for this one repo and for now I will avoid adding a second repo.
